protocol/avdl/chat1/local.avdl

```diff
@@ -256,8 +258,22 @@ protocol local {
     string senderUsername;
     string senderDeviceName;
     string senderDeviceType;
+
+    // MessageBoxed.V1: Hash of the encrypted header ciphertext.
+    // MessageBoxed.V2: Hash of MessageBoxed.headerSealed (.v || .n || .b)
     Hash headerHash;
+
+    // TOOD Maybe get rid of this field in favor of verificationKey.
+    //      If so, bump-nuke the caches in storage_blockengine and any other persistent
+    //      storage of MessageUnboxedValid.
+    // MessageBoxed.V1: Header signature. Included for the verification key.
+    // MessageBoxed.V2: Null
     union {null, SignatureInfo} headerSignature;
+
+    // MessageBoxed.V1: Null
+    // MessageBoxed.V2: The verification key used to unbox.
+    union {null, bytes} verificationKey;
```

Comment:
Will do. Also I misspoke. I was describing `MessageBoxed.HeaderVerificationKey`. This `MessageUnboxedValid.VerificationKey` is exactly the same key. But its reasons for being saved are:
- re-evaluating whether the device that signed the message gets revoked at any later time
- it seems like good record to keep